dotcom-rendering/package.json

```diff
@@ -50,7 +50,7 @@
 		"@guardian/libs": "21.4.0",
 		"@guardian/ophan-tracker-js": "2.2.5",
 		"@guardian/react-crossword": "2.0.2",
-		"@guardian/react-crossword-next": "npm:@guardian/react-crossword@0.0.0-canary-20250226111729",
+		"@guardian/react-crossword-next": "npm:@guardian/react-crossword@0.0.0-canary-20250303163323",
```

Comment:
Maybe worth mentioning here, this bump also incorporates the [fix to the "clear word" button](https://github.com/guardian/csnx/pull/1979) that prevents crossers from being clearer.

Reply:
Good shout 🙏 I've updated the description to include that for the benefit of future travellers.